### ISSUE TITLE
Repetition of data in different samples while trying the example in production Part of documentation

### ISSUE THREAD

**Issue by JosianeUwineza** [ElementAI/baal#200]
**Describe the bug**
Firstly thank you for this library.
I am having an issue while trying the example in the production Part of the documentation,  initial I selected 1000 images randomly from the training set when looping with to select the most uncertain samples some data are repeating for different iterations(https://baal.readthedocs.io/en/latest/notebooks/baal_prod_cls.html). 
And I though this is coming from this line:
labels = [get_label(train_dataset.files[idx]) for idx in top_uncertainty].
My question is this:
top_uncertainty indexes are for train_dataset? or are they for pool data?
If they are from the pool data and the labels are being obtained from the train dataset it is possible to repeat it. I do not know if I am right but your help on this will be much appreciated.

Thank you.

**Comment by Dref360** [comment]
Hello,

You are right. To get the right label, we need to "remap" them to the full dataset.


```python
# 4. Label those samples.
oracle_indices = active_learning_ds._pool_to_oracle_index(top_uncertainty)
labels = [get_label(train_dataset.files[idx]) for idx in oracle_indices]

active_learning_ds.label(top_uncertainty, labels)
```

`active_learning_ds.label` still uses `top_uncertainty` as it works with pool indices.

**Comment by JosianeUwineza** [comment]
@Dref360 thank you for the quick response, now it's working well.

**Comment by Dref360** [comment]
The updated notebook will be included on the documentation website ([baal.readthedocs.io/en/latest/notebooks/baal_prod_cls.html](https://baal.readthedocs.io/en/latest/notebooks/baal_prod_cls.html)). If you find other instances where we are not clear about this, please let me know and I'll fix it as soon as possible.
